docker/r/entrypoint.sh

```diff
@@ -8,5 +8,9 @@ for var in ${VariableArray[*]}; do
     fi
 done
 
+# add a symlink to the work directory containing the project to the home
```

Comment:
maybe this comment can simply say: "add a symlink to the project directory in /home/rstudio"